src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -772,10 +773,21 @@ private void OnAsyncTaskStateChanged(DynamoScheduler sender, TaskStateChangedEve
                         long end = e.Task.ExecutionEndTime.TickCount;
                         var executionTimeSpan = new TimeSpan(end - start);
 
-                        Dynamo.Logging.Analytics.TrackTimedEvent(
-                            Categories.Performance,
-                            e.Task.GetType().Name,
-                            executionTimeSpan);
+                        if (Logging.Analytics.ReportingAnalytics)
+                        {
+                            var modifiedNodes = "";
+                            if (updateTask.ModifiedNodes != null && updateTask.ModifiedNodes.Any())
+                            {
+                                modifiedNodes = updateTask.ModifiedNodes
+                                    .Select(n => n.NickName)
+                                    .Aggregate((x, y) => string.Format("{0}, {1}", x, y));
```

Comment:
Good catch @ellensi `NickName` is not really useful if the user has modified it. Ideally, I should get the original name, but at the moment there is no easy way to get the original name.